photon.cpp

```diff
@@ -86,6 +86,8 @@ static int init_event_engine(uint64_t engine, uint64_t flags, const PhotonOption
 int __photon_init(uint64_t event_engine, uint64_t io_engine, const PhotonOptions& options) {
     if (options.use_pooled_stack_allocator) {
         use_pooled_stack_allocator();
+    } else {
+        use_default_stack_allocator();
```

Comment:
each vcpu will call photon_init, and use_pooled_stack_allocator is a global config.
This change may leads to user set-up pooled-stack-allocator and then unset. (by initialize another vcpu without setting use_pooled_stack_allocator. That is highly risky.